community/sustainability_effort.md

```diff
@@ -0,0 +1,55 @@
+# Argo Workflows Sustainability Effort
+
+Argo Workflows is seeking more community involvement and ultimately more [Reviewers and Approvers](https://github.com/argoproj/argoproj/blob/main/community/membership.md) to help keep it viable. 
+
+We are trying out a system in which users pledge a certain number of hours per week (average) and in return have their own PRs prioritized for review.
+
+Help is also needed for triaging new incoming bugs (prioritizing them with `P1`, `P2`, and `P3` labels).
+
+## Terms
+
+The Argo project currently has 4 designated [roles](https://github.com/argoproj/argoproj/blob/main/community/membership.md):
+- Member
+- Reviewer
+- Approver
+- Lead
+
+For those who focus on just one area of the code (such as UI), the `Reviewer` and `Approver` roles can optionally be scoped to just that area.
+
+Anybody in any of these roles can sign up to be a part of this Incentive System. Participants are expected to try to advance in roles. The minimum expectations are different depending on which role:
+- Member: should focus primarily on authoring PRs, should average a minimum of 8 hours per week; can also assist with triaging bugs
+- Reviewer: should author PRs and review PRs to move into the "Approver" role, should average a minimum of 2 hours per week of either PR review time or triaging bugs
+- Approver/Lead: should average a minimum of 2 hours per week of either PR review time or triaging bugs
+
+Note that the hours per week from above is an average over time, and it's fine to have weeks of no activity.
+
+Current roles for individuals can be found in [this document](https://github.com/argoproj/argoproj/blob/main/MAINTAINERS.md).
+
+Participants should join the [#argo-wf-contributors](https://cloud-native.slack.com/archives/C0510EUH90V) slack channel.
+
+### Finding code to work on
+
+If you have a business need, definitely feel free to work on that. Otherwise, you can find an Issue and assign it to yourself. If you're a new developer, one option is to pick one up that has the 
+label "good-first-issue". Otherwise, you can pick up an Issue labeled "P1" or "P2". These are the Issues deemed most important to the project.
+
+### Finding a PR to review
+
+Ideally, look for the Pull Requests which have no existing Assignee. An Assignee is a Reviewer/Approver/Lead who owns reviewing the PR. In the case of a Reviewer being the Assignee, the Reviewer 
+can request a review from one or more Approvers once they have approved the PR.
+
+### Bug Triaging
+
+We need to make sure that all new bugs are seen by somebody so that we can identify the highest priority ones. There are labels "P1", "P2", and "P3" that should be applied, in which "P1" is considered
```

Comment:
Should also clarify what other priorities mean?

Reply:
done. Also realized that we do have a "P0" label as well.